### DISCUSSION TITLE
What should i do next?

### DISCUSSION THREAD

**Discussion by ErickLimaS** [ErickLimaS/anime-website#71] [Ideas]
### Give me your thoughts!

I'm still trying to improving this website to make it the "ultimate anime website",

so tell me what it needs, what is missing, what could i have done better... 

Theres some erros and bugs, im looking into they, remember to alert me in the GitHub **Issues** section.

... and thanks for sharing! :smiley:

**Comment by abdul-kaiyum1** [comment]
I recommend adding an option to watch content dubbed in English language and any others if available

**Reply by ErickLimaS** [reply]
But it currently has dub options, like this episode: https://aniproject-dev.vercel.app/watch/9919?source=aniwatch&episode=1&q=blue-exorcist-1198?ep=18064&dub=true

You just have to change it on the anime/manga page or in account settings

**Reply by abdul-kaiyum1** [reply]
but if we add option that will be better. If you can please do it

**Reply by ErickLimaS** [reply]
add a option where?

**Reply by abdul-kaiyum1** [reply]
Sorry i just see there was an button for dubbed. Sorry aging sir. And i have an problem that something it didn't play video ( It's mostly happened in pc ). Like the streaming link gogoanime api give i can easily play in 1084p in my same wifi but whenever i try to play in the web it slows

**Reply by abdul-kaiyum1** [reply]
And in Manga page Trending not showing results. Other options are also same saying no results found

**Reply by ErickLimaS** [reply]
> And i have an problem that something it didn't play video ( It's mostly happened in pc ). Like the streaming link gogoanime api give i can easily play in 1084p in my same wifi but whenever i try to play in the web it slows

Where are you hosting the consumet API(gogoanime)? So in localhost it works and when hosted it doesnt? The problem here is the video quality? Or the episode doesnt start at all?

**Reply by abdul-kaiyum1** [reply]
I host in vercel. Sometimes it doesn't start.

**Reply by abdul-kaiyum1** [reply]
And this error shows. So many times 


![Screenshot_2024-06-11-23-04-25-815_com android chrome](https://github.com/ErickLimaS/anime-website/assets/131800826/ab764b96-b11a-48cd-8b8a-1323907cd3ae)

**Reply by ErickLimaS** [reply]
I just test it, on your hosted version. I think its just a specific anime problem. I saw it happening on Kimetsu no Yaiba. 

The episode i tested: https://aikotv.vercel.app/watch/1735?source=aniwatch&episode=2&q=naruto-shippuden-355?ep=7883&t=11&dub=true

It usually happens on Dubbed Episodes, so try to change the one you are trying to watch to Sub and see if it works.

**Reply by abdul-kaiyum1** [reply]
Okay thank you so much, and thanks for making this source. And I have a question? Like can I modify the frontend according to me change name etc, with proper credit of your source?

**Reply by abdul-kaiyum1** [reply]
In manga. 
![Screenshot_2024-06-11-23-41-17-791_com android chrome](https://github.com/ErickLimaS/anime-website/assets/131800826/2f51ea17-4f85-48c5-9f71-23f2f037281c)

**Reply by ErickLimaS** [reply]
> Like can I modify the frontend according to me change name etc, with proper credit of your source?

Sure you can!

But before that, get the latest version of the repo. On footer has a Credits Section with a link to my repo that I made for this pupose. Its commented, so you just need to _uncomment_ to give the proper credits. 👍

**Reply by ErickLimaS** [reply]
> In manga. 

yeah, the source im using doesnt have data for mangas on Search Page. Only animes.

**Reply by abdul-kaiyum1** [reply]
Okay when you have time please add it. Thank you again 💖

**Reply by abdul-kaiyum1** [reply]
I have a suggestion to add this button in video player to navigate to episodes
![Screenshot_2024-06-12-00-02-42-153_com brave browser_beta](https://github.com/ErickLimaS/anime-website/assets/131800826/32d2c5c5-b647-4117-b80b-450de466ebe0)

**Reply by ErickLimaS** [reply]
> I have a suggestion to add this button in video player to navigate to episodes ![Screenshot_2024-06-12-00-02-42-153_com brave browser_beta](https://private-user-images.githubusercontent.com/131800826/338708805-32d2c5c5-b647-4117-b80b-450de466ebe0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgyMzIyMzYsIm5iZiI6MTcxODIzMTkzNiwicGF0aCI6Ii8xMzE4MDA4MjYvMzM4NzA4ODA1LTMyZDJjNWM1LWI2NDctNDExNy1iODBiLTQ1MGRlNDY2ZWJlMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxMlQyMjM4NTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMjA2NWEyZmZkZDQyZTA5ZTY1NGNkN2FmNTdiOTYzODA1YmYxOWRhZGI5M2M1ZjY2MTJlNjY4MjA4YTA2ODkxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.-ftl_N6b3AGFYZpggZyMMk8ycVzFf5k-rqE5hORw9Hk)

Good suggestion. I'll see what i can do.
